notebooks/elasticsearch-spark-recommender.ipynb

```diff
@@ -1114,6 +1148,24 @@
     "\n",
     "As you did with the similar movies recommendations, feel free to play around with the various queries you could pass into the user recommendation query."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5(c) Get users similar to another\n",
+    "\n",
+    "It is also possible to compute similar users for a given one. We have only indexed IDs for our users, but it is already functional enough."
```

Comment:
How about something like "It is also possible to compute similar users for a given one. Note that we have only indexed IDs for our users, so the results are difficult to interpret. However, we include the computation here for completeness."